recipes/rust-activation/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set version = "1.46.0" %}
+
+{% if cross_target_platform is undefined %}
```

Comment:
Shouldn't this be always defined?

Reply:
It should, but for some phases of conda-build is not, and this is a workaround.